.github/workflows/cmake.yml

```diff
@@ -16,39 +16,24 @@ jobs:
         - compiler_driver: g++
           compiler_prefix: /usr/bin
     steps:
-    - id: get-sha
-      run: echo ::set-output name=sha::$( curl https://api.github.com/repos/kokkos/mdspan/git/ref/heads/stable | jq .object.sha | tr -d '"' )
- 
-    - name: Determine whether mdspan needs to be rebuilt
-      id: cache-mdspan
-      uses: actions/cache@v2
-      with:
-        path: mdspan-install
-        key: mdspan-stable-${{ steps.get-sha.outputs.sha }}
- 
     - name: Create Build Environment
-      if: steps.cache-mdspan.outputs.cache-hit != 'true'
       run: cmake -E make_directory mdspan-build
       
     - name: Check Out
-      if: steps.cache-mdspan.outputs.cache-hit != 'true'
       uses: actions/checkout@v2
       with:
         repository: kokkos/mdspan
         path: mdspan-src
```

Comment:
Is there a way to ask for a specific branch or tag of mdspan here?  We don't immediately need that feature, but it could be useful in the future.  We could just set it to "stable" (lol) for now.